tests/ui/apps/Picker/Picker-View.js

```diff
@@ -0,0 +1,22 @@
+import Picker from '../../../../Picker';
+import ThemeDecorator from '../../../../ThemeDecorator';
+import React from 'react';
+
+const pickerList = [
+	'Romania',
```

Comment:
Hm... how about we replace these with something else? like fruits or ... 😁
Or we could reorder the list with A-Z.

Reply:
A-Z it will be 👍

Reply:
A-Z fruits are the choice 🍏 🍌 🟠